flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/fs/hdfs/HadoopRecoverableFsDataOutputStream.java

```diff
@@ -53,7 +56,9 @@
 @Internal
 class HadoopRecoverableFsDataOutputStream extends RecoverableFsDataOutputStream {
 
-    private static final long LEASE_TIMEOUT = 100_000L;
+    private static final long LEASE_RECOVERY_TIMEOUT_MS = 9_000_000L;
```

Comment:
Reference habse,
I guess you want to set 'LEASE_RECOVERY_TIMEOUT_MS'  to  `9_00_000L`(15 minute),  not `9_000_000L`

Reply:
Good catch! Thanks. @aubdiy

Reply:
@link3280  Are you Chinese?   I have some questions. Can I communicate with Wechat or QQ or nails?

Reply:
You can contact me at wechat link3280.

Reply:
@link3280    i have added you ,   and  Wrote a blog about this issue.

Reply:
@aubdiy Your blog is very informative. Thanks. But I didn't receive your friend request. Could you check if it succeeded?